lightkurve/targetpixelfile.py

```diff
@@ -831,6 +833,84 @@ def get_bkg_lightcurve(self, aperture_mask=None):
                           flux=np.nansum(self.flux_bkg[:, aperture_mask], axis=1),
                           flux_err=self.flux_bkg_err)
 
+
+    def _get_hdu(image, extension):
+        """Checks if an image is a single FITS image, or a FITS file,
+        and returns the appropriate PrimaryHDU.
+
+        Attributes
+        ----------
+        image : fits.ImageHDU or fits.HDUList object
+            FITS filename path or ImageHDU object to get the data from.
+        extension: integer
+            Number of the file extension containing data, depends on the file
+            type and source.
+
+        Returns
+        -------
+        hdu : astropy.PrimaryHDU
+            PrimaryHDU of the image inputted.
```

Comment:
It looks like this function can either return `PrimaryHDU` or `ImageHDU`, the docstring should clarify that.